src/PanelTraits/Update.php

```diff
@@ -73,4 +75,31 @@ public function getUpdateFields($id)
 
         return $fields;
     }
+
+    /**
+     * Get the value of the 'name' attribute from the declared relation model in the given field.
+     *
+     * @param Model $entry The model.
+     * @param array $field The CRUD field array.
+     *
+     * @return mixed The value of the 'name' attribute from the relation model.
+     */
+    private function getEntryValue($entry, $field)
```

Comment:
Not sure ```getEntryValue()``` is the best name for this method. It could be confused with getting the value of the entry - as in, all values of the entry. Looks like a getter for ```$this->crud->entry``` to me, at first glance, and I think that might be confusing in the long run.

What do you think of:

- ```getFieldValue()```
- ```getEntryFieldValue()```
- ```getAttributeValue()```
- ```getEntryAttributeValue()```

? Smth like this?